reports/api.py

```diff
@@ -125,11 +125,15 @@ def get_languages_spoken_at_home(self, obj: Child) -> List[Optional[str]]:
 
 @extend_schema_view(list=extend_schema(description="Get all children data."))
 class ChildViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
-    queryset = Child.objects.exclude(guardians=None).prefetch_related(
-        Prefetch(
-            "guardians__languages_spoken_at_home",
-            queryset=Language.objects.order_by("alpha_3_code"),
-            to_attr="prefetched_languages_spoken_at_home",
-        ),
+    queryset = (
+        Child.objects.exclude(guardians=None)
+        .prefetch_related(
+            Prefetch(
+                "guardians__languages_spoken_at_home",
+                queryset=Language.objects.order_by("alpha_3_code"),
+                to_attr="prefetched_languages_spoken_at_home",
+            ),
+        )
+        .order_by("last_name", "first_name")
```

Comment:
Is this a fix for [KK-953](https://helsinkisolutionoffice.atlassian.net/browse/KK-953)?

Reply:
most probably

Reply:
its here also https://github.com/City-of-Helsinki/kukkuu/pull/293/files#diff-58d75c954f04eb71a49cfbd1338dca4b196bc590b6e2ae2b424f1f23416519bfR24

Reply:
Yep, looks like the data in the bug report KK-953.